interactions-transitions-assignment/part-1-bar-chart/us-emissions-interaction.js

```diff
@@ -21,11 +22,138 @@ function createChart(elementId) {
         .append('g')
         .attr('class', 'svg-group')
         .attr('transform', 'translate(' + margins.left + ',' + margins.top + ')');
+  // read in air quality data
+  d3.csv('air_quality.csv').then(function(data) {
```

Comment:
You can get rid of the "function(data)" here and replace it with "(data) =>." Same with your other uses of similar syntax below